app/src/main/java/com/truongdc21/mediatree/intro/FirstInstallActivity.kt

```diff
@@ -3,14 +3,19 @@ package com.truongdc21.mediatree.intro
 import com.truongdc21.mediatree.MainActivity
 import com.truongdc21.mediatree.base.BaseActivity
 import com.truongdc21.mediatree.databinding.ActivityFirstInstallBinding
+import com.truongdc21.mediatree.ui.activity.LoginActivity
 import com.truongdc21.mediatree.utils.switchActivity
 class FirstInstallActivity : BaseActivity<ActivityFirstInstallBinding>(ActivityFirstInstallBinding::inflate){
+
     override fun initViews() {
         binding.apply {
             btnSkips.setOnClickListener {
                 this@FirstInstallActivity.switchActivity(MainActivity())
```

Comment:
```suggestion
                this@FirstInstallActivity.switchActivity(MainActivity.newInstance())
```
Dùng cái này để có lúc truyền bundle ông nhé